bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManagerImpl.java

```diff
@@ -599,13 +607,20 @@ private void initializeHandler(Thing thing) {
             }
             ThingType thingType = getThingType(thing);
             applyDefaultConfiguration(thing, thingType);
-            if (isInitializable(thing, thingType)) {
-                setThingStatus(thing, buildStatusInfo(ThingStatus.INITIALIZING, ThingStatusDetail.NONE));
-                doInitializeHandler(thing.getHandler());
+
+            if (isPersistedAsEnabled(thing.getUID())) {
+                if (isInitializable(thing, thingType)) {
+                    setThingStatus(thing, buildStatusInfo(ThingStatus.INITIALIZING, ThingStatusDetail.NONE));
+                    doInitializeHandler(thing.getHandler());
+                } else {
+                    logger.debug("Thing '{}' not initializable, check required configuration parameters.",
+                            thing.getUID());
+                    setThingStatus(thing, buildStatusInfo(ThingStatus.UNINITIALIZED,
+                            ThingStatusDetail.HANDLER_CONFIGURATION_PENDING));
+                }
             } else {
-                logger.debug("Thing '{}' not initializable, check required configuration parameters.", thing.getUID());
-                setThingStatus(thing,
-                        buildStatusInfo(ThingStatus.UNINITIALIZED, ThingStatusDetail.HANDLER_CONFIGURATION_PENDING));
+                // the thing is persisted as disabled. Will not attempt to initialize its handler.
+                setThingStatus(thing, new ThingStatusInfo(ThingStatus.UNINITIALIZED, ThingStatusDetail.DISABLED, null));
```

Comment:
It should have been in exactly this status before already - so no need to set it here.